docs/Rakefile

```diff
@@ -112,15 +112,18 @@ task :adoc_variables => :init do
   centos_iso_version   = ENV['CENTOS_ISO_VERSION']   || 'unset'
   minikube_iso_version = ENV['MINIKUBE_ISO_VERSION'] || 'unset'
 
-  ADOC_VARIABLES.each do |f|
-    File.open(f, "w+") do |f|
-      f.write(":project: Minishift\n")
-      f.write(":minishift-version: #{minishift_version}\n")
-      f.write(":openshift-version: #{openshift_version}\n")
-      f.write(":centos-iso-version: #{centos_iso_version}\n")
-      f.write(":minikube-iso-version: #{minikube_iso_version}\n")
-    end
-  end
+  contents = <<EOF
+:project: Minishift
+:download-url: https://github.com/minishift/minishift/releases
+:download-text: {project} Releases
```

Comment:
@robin-owen What is the need to add `download-text` and `download-url` as a dynamic variable? I do not think it is necessary.

Reply:
@LalatenduMohanty It's used in more than one document. I thought that it made more sense to have the attributes in one location instead of two or more which would need to be manually maintained later.

Reply:
@LalatenduMohanty I would prefer it if we could create a single file within the documentation to keep such attributes in, but for the time being, this seemed like the most logical choice.

Reply:
Sounds good to me